### PR TITLE
Script to apply configlet added to /usr/bin in image.

### PR DESCRIPTION
**Pull request by renukamanavalan**
**- What I did**
SONiC requires the ability to apply configlets.
This script is built on top of ConfigDBConnector
This is a very simple script that parses and invokes API from ConfigDBConnector to apply the updates.
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by renukamanavalan** [comment]
retest vs please.

**Comment by renukamanavalan** [comment]
retest vsimage please

**Comment by renukamanavalan** [comment]
retest this please.

File: build_debian.sh [hunk 1]
```diff
@@ -446,6 +446,9 @@ EOF
 ## Copy over clean-up script
 sudo cp ./files/scripts/core_cleanup.py $FILESYSTEM_ROOT/usr/bin/core_cleanup.py
 
+## Copy over configlet application script
+sudo cp ./files/image_config/configlet/configlet.py $FILESYSTEM_ROOT/usr/bin/configlet.py
```
Comment (lguohan) [inline build_debian.sh]: suggest to move this into sonic utilities

**Comment by renukamanavalan** [comment]
Moved to sonic-utilities and added to setup.py, which would copy to /usr/bin. Hence this PR is no longer needed.

**Pull request closed without merging**